### PR TITLE
remove dropdown

### PR DESCRIPTION
**Pull request by tac0turtle**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -105,20 +105,9 @@ const config = {
...
-            label: 'Build',
-            position: 'left',
-            items: [
-              {
-                type: "doc",
-                label: "Build a Module",
-                docId: "build/building-modules/intro",
-              },
-              {
-                label: "REST API",
```
Comment (julienrbrt) [inline docusaurus.config.js:118]: Should we this keep somewhere?

File: docusaurus.config.js [hunk 2]
```diff
@@ -105,20 +105,9 @@ const config = {
...
-                to: "api",
-              },
-            ],
+            type: "doc",
+            label: "Build a Module",
```
Comment (julienrbrt) [inline docusaurus.config.js:109]: I would just say build

**Review by julienrbrt — APPROVED** [review]
utACK

File: docusaurus.config.js (hunk 1 above)
Comment (tac0turtle) [reply docusaurus.config.js:118]: added at the end of the navbar

**Pull request merged**